### PR TITLE
fix createDaoFactory in wherehows-frontend

### PR DESCRIPTION
**Pull request by camelliazhang**

### CODE REVIEW

File: wherehows-frontend/app/controllers/Application.java [hunk 1]
```diff
@@ -43,15 +48,43 @@
     private static final String PIWIK_URL = Play.application().configuration().getString("tracking.piwik.url");
     private static final Boolean IS_INTERNAL = Play.application().configuration().getBoolean("linkedin.internal", false);
 
+    private static final String DB_WHEREHOWS_URL = Play.application().configuration().getString("db.wherehows.url");
```
Comment (mars-lan) [inline wherehows-frontend/app/controllers/Application.java]: Should be a different value for frontend?

**Comment by camelliazhang** [comment]
Change to use frontend opensource configs

**Comment by mars-lan** [comment]
Please refer to https://github.com/linkedin/WhereHows/pull/641 for relevant changes

**Comment by mars-lan** [comment]
Not needed any more

**Pull request closed without merging**